docs/readingindata.rst

```diff
@@ -11,6 +11,14 @@ This will include 1D and 2D spectra, images, data tables, and associated images.
 This page documents how to read in MOS data from another observatory.
 An example is shown at the bottom (:ref:`reading_in_data`).
 
+The general workflow can be summarized as follows:
+
+1. Start Glue by typing `glue` in the command line
```

Comment:
Use double ticks on each side, or Sphinx is going to try link that to some API and fail.